spec/classes/database_spec.rb

```diff
@@ -11,8 +11,14 @@
     next if facts[:os]['name'] == 'windows'
 
     context "on #{os}" do
+      systemd_fact = case facts[:os]['family']
```

Comment:
We have this mocked if the module is present. You'll need https://github.com/voxpupuli/modulesync_config/commit/607bf0021177cd16e50d902cdba7e82a8b381213 but that's in since 10ce29fe3f9a99fc2dc3c652e0f2ad3675a11c87 so it should already be present.